packages/langium/package.json

```diff
@@ -19,6 +19,10 @@
   "files": [
     "lib",
     "src",
+    "browser.js",
+    "browser.d.ts",
+    "node.js",
+    "node.d.ts",
```

Comment:
It could become this also this:
```
  "main": "lib/node/index.js",
  "types": "lib/node/index.d.ts",
    ".": {
      "types": "./lib/node/index.d.ts",
      "default": "./lib/node/index.js"
    },
    "./browser": {
      "types": "./lib/browser/index.d.ts",
      "default": "./lib/browser/index.js"
    }
  },
  "typesVersions": {
    "*": {
      "browser": [ "lib/browser" ]
    }
  },
```

Reply:
> then the default exports links directly to node

I don't think that's a good idea, since we want to import from `langium` for runtime independent code, that we can later reuse for `langium/node` and `langium/browser` based language servers.

But otherwise, I agree 👍